data-access-layer/eventmesh/LockManager.js

```diff
@@ -131,19 +127,28 @@ class LockManager {
             resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.LOCK,
             resourceType: CONST.APISERVER.RESOURCE_TYPES.DEPLOYMENT_LOCKS,
             resourceId: resourceId,
-            options: opts
+            metadata: resource.metadata,
```

Comment:
while updating shouldn't we send resource version as well? Looked at the code in apiserver.unlock and not sure how that is handling with versions? Because thats what we are to use to avoid concurrent writes.

Reply:
resourceVersion is part of metadata.